app/static/js/admin/event/wizard/vue-app.js

```diff
@@ -176,6 +179,10 @@ var app = new Vue({
         },
         addTicket: function (ticketType) {
             var ticket = _.cloneDeep(TICKET);
+            ticket.sales_start_date = moment().tz(this.event.timezone).format('MM/DD/YYYY');
```

Comment:
@niranjan94  yes but without doing this you are using local PC time as the default. Suppose a user resides in delhi but is trying to create a ticket for an event in say london. This will take care of that as time shown will be according to london which is the exact issue #2805 .